modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/QpidAndesBridge.java

```diff
@@ -569,7 +569,7 @@ private static void addLocalSubscriptionsForAllBindingsOfQueue(AMQQueue queue, S
                     }
 
                     ProtocolType protocol = ProtocolType.AMQP;
-                    String subscriptionIdentifier = "";
+                    String subscriptionIdentifier = queue.getName();
```

Comment:
Isn't it better to rename this local variable to reflect that? or else we should at lease have a code comment explaining why we do this.